tmpl/wiki/faq.md

```diff
@@ -1,25 +1,24 @@
 *Q: Is MirageOS 'still' Linux? Does it require a Linux host on which to run?*
 
-MirageOS is a '[library operating system](http://anil.recoil.org/papers/2013-asplos-mirage.pdf)', which means that it can run on any target for which a suitable bootloader and drivers exist. The first two targets are Unix and Xen, but we have prototypes that compile the same application source code (e.g. the MirageOS website) to run as kernel modules inside FreeBSD, or even to JavaScript. MirageOS provides support to more easily target such diverse environments due to its emphasis on modular programming and compile-time specialisation. We'll have a series of blog posts over December with tutorials on how the FreeBSD/kernel and JavaScript targets work, to aid developers who wish to port other environments such as the Raspberry Pi, Docker, Linux kernel modules, or to other hypervisors such as KVM.
+MirageOS is a '[library operating system](http://anil.recoil.org/papers/2013-asplos-mirage.pdf)', which means that it can run on any target for which a suitable bootloader and drivers exist. The most mature targets are Unix and Xen.  Support for KVM via [solo5](/blog/introducing-solo5) is also fairly well-supported.  We have prototypes that compile the same application source code (e.g. the MirageOS website) to run as kernel modules inside FreeBSD, or even to JavaScript. MirageOS provides support to more easily target such diverse environments due to its emphasis on modular programming and compile-time specialisation.  If you'd like to dig into some code, see [the mirage-platform library](https://github.com/mirage/mirage-platform), which has the OCaml modules referred to in the Unix and Xen targets.
```

Comment:
it looks like (via https://github.com/mirage/mirage/issues/186): https://github.com/pgj/mirage-kfreebsd